src/float/add.rs

```diff
@@ -181,117 +181,30 @@ macro_rules! add {
     }
 }
 
+// NOTE(cfg) These are disabled for gnueabihf due to 
+// https://github.com/rust-lang-nursery/compiler-builtins/issues/90
+#[cfg(not(gnueabi))]
```

Comment:
So should the addsf3.c files be enabled for `arm-unknown-linux-gnueabi` since we're disabling the rust version?